azure-spring-boot-bom/pom.xml

```diff
@@ -46,6 +46,16 @@
     </scm>
 
```

Comment:
This BOM intended to provide dependency management for azure related packages only. For details, you can check issue #128.